e2e-test/cypress/integration/scoll_back_to_top_spec.js

```diff
@@ -0,0 +1,19 @@
+describe('Scroll Back To Top', () => {
+  before(function() {
+    cy.login("mstein", "test");
+    cy.visit(`/projects`);
+  });
+  it('When page load initially, the button back to top should not be visible ', () => {
+    cy.get("[data-test=backToTop-button]").should("not.be.visible");
+  });
+  it('On scroll, the button back to top should be visible ', () => {
+	  cy.scrollTo(0, 520);
+    cy.get("[data-test=backToTop-button]").should("visible");
+  });
+  it('On button click, scroll back to top ', () => {
+	  cy.get("[data-test=backToTop-button]").click();
+	  cy.focused().click();
+	  cy.scrollTo(0, 0);
```

Comment:
Should this `scrollTo` be here? Since we are clicking on button that is scrolling us up, there should be no need to scroll to top again.